landing/public/index.html

```diff
@@ -3,36 +3,220 @@
+        <a class="navOptions" href="pricing.html">Pricing</a>
+        <a class="navOptions" href="faqs.html">FAQs</a>
+        <button class = "navMobileButton primary-btn"><a class = "primary-btn-link">Join Early Access</a></button>
+      </div>
+  
+      <div class = "navButton">
+        <button class = "primary-white-btn"><a class = "primary-white-btn-link">Join Early Access</a></button>
+      </div>
+    </div>
+  </nav>
+
+  <img class = "heroCircle1" src = "static/background/heroCircle1.png">
+  <img class = "heroCircle2" src = "static/background/heroCircle2.png">
+  <img class = "heroCircle3" src = "static/background/heroCircle3.png">
+
+
+  <section class = "hero">
+    <!-- <img class = "heroBG" src = "static/background/heroBG.png"> -->
+    <div class = "heroText" data-aos = "fade-right" data-aos-delay: "2000">
+      <h1 class = "heroHead">Making your life easier, one order at a time.</h1>
+      <p class = "heroBody">Quicklink automates order and financial processes, management, and documentation for businesses like yours to increase efficiency and sales.</p>
+      <button class = "primary-white-btn"><a class = "primary-white-btn-link">Join Early Access</a></button>
+    </div>
 
-  <main class="landing">
-    <section class="home">
-      <div class="home-container">
-        <img src="static/logos/logo.png" alt="Quicklink Logo">
-        <p>coming soon</p>
+    <img class = "heroImage" src="./static/mockups/mockups.png" data-aos = "fade-left" data-aos-delay: "2000">
+  </section>
+
+  <section class = "vp1" data-aos = "fade-up">
+    <div class = "textArea">
+      <div class = "line"></div>
+      <div class = "vpText">
+        <h1>Selling made easier.</h1>
+        <p>You’re probably using multiple applications to sell your products, document transactions, and receive payments whenever you have a customer. Quicklink gets rid of this hassle by putting all of these processes into one platform — all you need to do is get started!</p>
       </div>
-    </section>
-  </main>
+    </div>
 
-  <script src="js/vendor/modernizr-3.11.2.min.js"></script>
-  <script src="js/plugins.js"></script>
-  <script src="js/main.js"></script>
+    <img class = "vp1Image" src = "./static/mockups/VP1.png">
+
+  </section>
+
+  <img class = "circleBack1" src = "static/background/vp2Circle.png">
+
+  <img class = "circleBack2" src = "static/background/vp2Circle2.png">
+
+  <section class = "customers">
+    <h1 class = "bold" data-aos = "fade-up">So, who are we for?</h1>
+
+    <div class = "customerCards">
+
+      <div class = "customerCard" data-aos = "fade-right">
+        <img src = "static/mockups/customers1.svg">
+
+        <div class = "customerText">
+          <h3>Online Food Businesses</h3>
+          <p>For small businesses selling meals, pastries, and other delicacies in Instagram, Facebook, and more!</p>
+          <button class = "secondary-btn"><a class = "secondary-btn-link">Read More</a></button>
```

Comment:
don't forget to `href=url`

if wala pa page na na create for the url, leave a comment nalang above the line para ma-remind

Reply:
same goes with the other `<a>` tags without hrefs